lower_bound: [150, 120, 100]
upper_bound: [150, 140, 150]
lower_bound_car: [120, 100, 100]
upper_bound_car: [130, 255, 255]
canny_lower: 50
canny_upper: 150
fov: 0.55
hough_rho: 6
hough_threshold: 50
hough_min_line_length: 20
hough_max_line_gap: 100
angle_lower_bound: 20
angle_upper_bound: 90
deviation_threshold: 2
create_visualisation: True